### PR TITLE
Update opensearch doc

### PR DESCRIPTION
**Pull request by ribaraka**
Updated OpenSearch documentation to API v2

### CODE REVIEW

File: doc/clusters/opensearch.md [hunk 1]
```diff
@@ -2,19 +2,56 @@
...
+| dataNodes                | Array of objects ([DataNodes](#DataNodes))                             | List of data node settings                                                                                                                                                                                                                                                                                                                   |
+| dashboards               | Array of objects ([Dashboards](#Dashboards))                           | List of dashboards settings                                                                                                                                                                                                                                                                                                                  |
```
Comment (RostislavPorohnya) [inline doc/clusters/opensearch.md:29]: dashboard node settings

File: doc/clusters/opensearch.md [hunk 2]
```diff
@@ -2,19 +2,56 @@
...
+
+### DataNodes 
+
+| Field               | Type                                        | Description       |
+|---------------------|---------------------------------------------|-------------------|
+| nodeSize            | String   <br /> **required**<br />_mutable_ | Size of data node |
+| nodesNumber         | Int32  <br /> **required**  _mutable_       | Number of nodes   |
```
[the remark this answers is not in the document]
Comment (ribaraka) [reply doc/clusters/opensearch.md:36]: fixed

File: doc/clusters/opensearch.md (hunk 1 above)
Comment (ribaraka) [reply doc/clusters/opensearch.md]: fixed

File: doc/clusters/opensearch.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ribaraka) [reply doc/clusters/opensearch.md]: added + replaced `<br />` by `<br>`

File: doc/clusters/opensearch.md [hunk 3]
```diff
@@ -2,49 +2,77 @@
...
+| name                     | String <br> **required**                                               | Cluster name. Should have length from 3 to 32 symbols.                                                                                                                                                                                                                                                                                       |
+| version                  | String <br> **required**                                               | OpenSearch instance version.                                                                                                                                                                                                                                                                                                                 |
```
Comment (testisnullus) [inline doc/clusters/opensearch.md:8]: String -> string, Int -> int...

**Review by testisnullus — CHANGES_REQUESTED** [review]

File: doc/clusters/opensearch.md (hunk 3 above)
Comment (ribaraka) [reply doc/clusters/opensearch.md]: fixed

**Review by testisnullus — APPROVED** [review]

**Pull request merged**